bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -52,13 +53,17 @@ private static void csrReply(PrintWriter writer) {
     private static void jbsReply(PullRequest pr, PrintWriter writer) {
         writer.println("@" + pr.author().username() + " this pull request must refer to an issue in " +
                       "[JBS](https://bugs.openjdk.org) to be able to link it to a [CSR](https://wiki.openjdk.org/display/csr/Main) request. To refer this pull request to " +
-                      "an issue in JBS, please use the `/issue` command in a comment in this pull request.");
+                      "an issue in JBS, please update the title of this pull request to just the issue ID.");
+    }
+
+    private static void multipleIssueLinkReply(PullRequest pr, PrintWriter writer) {
```

Comment:
This reply isn't linking to anything so I would suggest:
```suggestion
    private static void multipleIssueReply(PullRequest pr, PrintWriter writer) {
```